CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 1.1.3 (December 12, 2019)
```

Comment:
I'd say new authentication method desires **minor** version number, not a patch one.
And an actual date
```suggestion
# 1.2.0 (December 18, 2019)
```